<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Console</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PhoneBook</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>203</width>
       <height>428</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_firstName">
        <property name="font">
         <font>
          <family>Lucida Console</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_firstNmae"/>
      </item>
      <item>
       <widget class="QLabel" name="label_LastName">
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_lastName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_dateOfBirth">
        <property name="text">
         <string>Дата рождения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="input_dateOfBirth"/>
      </item>
      <item>
       <widget class="QLabel" name="label_addressCountry">
        <property name="text">
         <string>Страна</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_addressCountry"/>
      </item>
      <item>
       <widget class="QLabel" name="label_addressRegion">
        <property name="text">
         <string>Область</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_addressRegion"/>
      </item>
      <item>
       <widget class="QLabel" name="label_addressCity">
        <property name="text">
         <string>Город</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_addressCity"/>
      </item>
      <item>
       <widget class="QLabel" name="label_addressStreet">
        <property name="text">
         <string>Улица</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_addressStreet"/>
      </item>
      <item>
       <widget class="QLabel" name="label_addressHouse">
        <property name="text">
         <string>Дом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_addressHouse"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="button_save">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_clear">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>10</y>
      <width>258</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Title">
         <property name="font">
          <font>
           <family>Lucida Console</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="text">
          <string>Справочник</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QColumnView" name="columnView">
       <property name="resizeGripsVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
